data/application.css

```diff
@@ -176,14 +176,35 @@ row:hover .topic {
     padding: 4px;
 }
 
-row:selected .topic {
+row:selected .topic:not(.insensitive) {
     background-color: #f18c22;
 }
 
-row:selected:hover .topic {
+row:selected:hover .topic:not(.insensitive) {
     background-color: #ffa63c;  /* lighten #f18c22 10% */
 }
 
+.topic.insensitive {
+    background-color:  rgba(0, 0, 0, 0.3);
+    border-radius: 25px;
```

Comment:
These lines that apply to both sensitive and insensitive topics can be removed, as `.topic.insensitive` inherits and overrides rules from `.topic`.